base.py

```diff
@@ -143,6 +143,7 @@ def load_csv(
             # store episode_id to group_per_episode
             self.episode_ids = df[episode_col].values
 
+        self.diff_state = diff_state
         if diff_state == True:
             logging.info(
                 "delta states enabled, calculating differential between current and previous predicted output"
```

Comment:
Thanks for the PR @juan. Here this could simply be "delta states enabled, calculating differential between input and output states". The input may not be previous predicted output for single step approach.

Reply:
In the meantime I will change this to "delta states enabled, calculating differential between input and output values" -- avoiding 'states', to avoid confusion between inputs/outputs and states/actions.

Reply:
The delta state change is s_t+1-s_t, actions are allowed to be independent, at least in the cases I was considering. But, using input/output is okay.